src/main/lib/sheet-utils.js

```diff
@@ -0,0 +1,20 @@
+const groupToSheets = paths => {
+  return paths.reduce((groupMap, filePath) => {
+    // Split on the first '/'
+    const splitPath = filePath.split(/\/(.+)/, 2);
+    const sheetName = splitPath[0];
+
+    if (!Object.prototype.hasOwnProperty.call(groupMap, sheetName)) {
```

Comment:
Can you just check if it is defined without using `Object.prototype.hasOwnProperty` machine gun?

For me `groupMap[sheetName] === undefined` is way simpler.

Reply:
IDE doesn't seem to complain about it and it is simpler so changed to suggestion